blackjack/api/views.py

```diff
@@ -8,7 +9,15 @@
 from rest_framework.generics import ListAPIView, RetrieveAPIView, get_object_or_404
 
 
+class Game:
+    deck_cards = [2, 2, 2, 2, 3, 3, 3, 3, 4, 4, 4, 4, 5, 5, 5, 5, 6, 6, 6, 6, 7, 7, 7, 7, 8, 8, 8, 8, 9, 9, 9, 9,
+                  10, 10, 10, 10, 'J', 'J', 'J', 'J', 'Q', 'Q', 'Q', 'Q', 'K', 'K', 'K', 'K', 'A', 'A', 'A', 'A'
+                  ]
 
-@api_view(["GET"])
-def deal(request):
-    return JsonResponse(data={'key':'value'}, status=status.HTTP_200_OK)
\ No newline at end of file
+    def __init__(self):
+        random.shuffle(self.deck_cards)
+
+    @api_view(["GET"])
+    def deal(self, request):
```

Comment:
I wouldn't make this a method on the Game class. Drop this back to a regular function (no `self`), and instantiate an instance of Game before dealing a card.

Reply:
that worked! thank you! 

as a general rule, are endpoints not methods on a class even if it is a function of the class? does that make sense?